.dockerignore

```diff
@@ -1,6 +0,0 @@
-﻿.git
-.github
-.gitmodules
-.dockerignore
-.editorconfig
-Containerfile
```

Comment:
Any reason why you chose to delete this file?

Reply:
Neither the GitHub CI Actions nor the way how the local container deployment works require a `.dockerignore` file.

```bash
docker build -t docs -f Containerfile .
```

There is only one file copied to the container image `COPY [ "requirements.txt", "./"]` and everything else is mounted as a RO volume in the subsequent `docker run ...` command. Therefore I removed the file or am I not seeing something obvious?

Reply:
When you run docker manually it copies everything, and .gitignore isn't helping. Careful use of Dockerfile and docker-compose.yml probably ensures that not the whole folder contents is copied + therefore used as basis to decide when rebuild is needed. I suggest keeping the dockerignore and having it list log files and unrelated files. I might've recommended this as well once when grin was very privacy focused, as any logfiles, rust binaries with local user folder names, and similar could risk leaking in to the docker image, and then leak out as uninitialized memory. I think something such happened once, just not - known to me - in combination to leak any PII.

Reply:
@sesam This `Containerfile` is only used for debugging purposes to test the website locally without pushing everthing to master just to see if things work but if the performance of the build process is too slow, because the context copied is too big or PII leaks are real, than I will include the `.dockerignore` file again.